plugin/pom.xml

```diff
@@ -64,5 +72,66 @@
             <version>3.0.1</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>scm-api</artifactId>
+            <version>1.3</version>
```

Comment:
Things like this you probably want in `dependencyManagement`, since they are not direct dependencies, pending [JENKINS-47498](https://issues.jenkins-ci.org/browse/JENKINS-47498).

Reply:
Done.